src/test/java/hw5/pages/UserTablePage.java

```diff
@@ -0,0 +1,98 @@
+package hw5.pages;
+
+import hw5.pages.composites.LogsList;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class UserTablePage extends AbstractPage {
+
+    public UserTablePage(WebDriver driver) {
```

Comment:
What is the proper place for constructor in class according to **Java Code Convention**?